src/test/ui/type-alias-impl-trait/issue-69323.rs

```diff
@@ -0,0 +1,19 @@
+// check-pass
+
+// revisions: min_trait full_trait
```

Comment:
This is a good idea. But let's just do full/min.

Reply:
Other tests use `min/full_tait` and I picked them up (actually, I typoed though). Anyway fixed.